draft-ietf-quic-http.md

```diff
@@ -841,6 +841,22 @@ A server MUST treat a MAX_PUSH_ID frame payload that is other than 4 octets in
 length as a connection error of type HTTP_MALFORMED_MAX_PUSH_ID.
 
 
+# Connection Management
+
+QUIC connections are persistent.  All of the considerations in Section 9.1 of
+{{?RFC7540}} apply to the management of QUIC connections.
+
+HTTP clients are expected to use QUIC PING frames to keep connections open.
+Servers SHOULD NOT use PING frames to keep a connection open.  A client SHOULD
+NOT use PING frames for this purpose unless there are no responses outstanding
```

Comment:
I think you have one more negative here than you meant.  A client SHOULD NOT use PING frames for this purpose when there are no responses outstanding.  If you're waiting on a server response, that's exactly when it's critical to keep the connection alive and the NAT bindings fresh, so that you're sure you can get the server response when it finally arrives.  If you're not waiting on anything, that's when it's fine to let the connection idle and potentially close.